inference_rules.adoc

```diff
@@ -428,6 +428,79 @@ division allows calibration for quantization and does not allow any retraining.
+1. Be highly scalable, reliable, and fault-tolerant
+
+Examples of forbidden bus interconnect include: PCIe/CXL/CCIX, Hypertransport, NVLink, QPI, UPI, and ICI (interchip interconnect).
+
+The SUT and QDL must run on two physically separated and distinct systems.
+
+Fabric and protocol must be reported in the submission metadata. Submission metadata must be include sufficient to determine OSI layers one through four of the submission's network stack.
+
+==== SUT over the Network and Setup Constraints
+
+* SUT parameters and configuration must be specified in detail in the submission results. 
+* Network interface characteristics must be added to the report, both at the SUT side and at the loadgen server side, including networking components or any other hardware in the system that either stands between them or are involved in any way in the test.
+
+==== Benchmarks and QSL Preprocessing
+
+Current benchmark scenarios remain the same but tested over the network. As specified below the data formats for inputs and outputs can be adapted and preprocessed for network transmission, providing a tradeoff between compute and network bandwidth. The preprocessing must be selected only from the options at the table below. The selected option can be defined by the submitter according to the specific network link and acceleration implemented. The preparation of the data is done in the QSL in an untimed manner. The details for the allowed preprocessing options for each benchmark are specified in the table below.
+
+The table below provides the allowed pre-processing and post-processing options for each benchmark.
+|===
+|*Area* | *Task* | *Model* | *QSL side PreProcessing^(1,2,3)^*
+|Vision | Image classification | Resnet50-v1.5 | Allow one of the following compression options for pre-processing:
+
+1) No compression 2) Lossless compression 3) The original compression of the dataset (JPEG)
+|Vision | Object detection (large) | SSD-ResNet34 Going to change to Retinanet-resnext50 | Allow one of the following compression options for pre-processing:
+
+1) No compression 2) Lossless compression 3) The original compression of the dataset (For the Coco dataset JPEG, for Open Images JPEG) 
+|Vision | Medical image segmentation | 3D UNET | Allow one of the following compression options:
+
+1) No compression 2) Lossless compression
+
+This rule applies both for the QSL pre-processing and for post-processing function allowed in QDL for this benchmark results.
+|Speech | Speech-to-text | RNNT | Allow one of the following compression options for pre-processing:
+
+1) No compression 2) Lossless compression 3) The original compression of the dataset (FLAC)
+|Language | Language processing | BERT-large | Text in. No compression allowed.
+|Commerce | Recommendation | DLRM | QDL sends query (Batch of samples).
+
+Allow one of the following compression options for pre-processing:
+
+1) No compression 2) Lossless compression
+
+Allow any reasonable lossless compression.
```

Comment:
what does "reasonable" mean here?

Reply:
Allow lossless compression that will be suitable for production use.